### ISSUE TITLE
Replace the EAN code of the offer One Fulfillment

### ISSUE THREAD

**Issue by yao657355700** [allegro/allegro-api#9494]
For example:
There is now an offer with EAN code 123456, and the inventory in the Polish warehouse is 30 units.
I am now requesting to change the EAN code of the offer to 987654321 through a report error,
But the barcode on the 30 inventory items in the warehouse is still the old EAN code: 123456,
So can these 30 inventories still be sold and shipped normally?

**Comment by MartaNowaczyk** [comment]
This should not affect product sales and shipments.

**Comment by yao657355700** [comment]
The EAN code of the offer was changed to: 987654321
But the existing inventory in the warehouse was labeled with the old EAN code (123456).

After the offer was purchased, how did you find the product with the EAN code (123456) in the warehouse?

**Comment by yao657355700** [comment]
For example, my offer
OfferID: 15619724144
Its old EAN code is: 768434035332, and there are 8 stocks in the warehouse.
The EAN code on these 8 stocks is: 768434035332.

Now the EAN code of this offer has been updated to: 6977631510892.

What should I do with these 8 stocks?
After the customer purchases, can the warehouse still find this product for delivery?
![333](https://github.com/user-attachments/assets/a09aa376-ff96-49f2-818c-883e562f2245)

**Comment by MartaNowaczyk** [comment]
I think it will not be a problem for the warehouse to handle it. These are not API issues, so if you want more details write via the [contact form](https://allegro.pl/pomoc/kontakt).

**Comment by yao657355700** [comment]
> I think it will not be a problem for the warehouse to handle it. These are not API issues, so if you want more details write via the [contact form](https://allegro.pl/pomoc/kontakt).

I contacted customer service through the form.
But each of them gave a different answer.

So I want to ask you about your technology, what is the logic of the system?

**Comment by yao657355700** [comment]
The manager in charge of One Fulfillment told me that I had to wait until all the inventory was sold out before I could update the EAN code. However, sometimes when I edited a product, the system would prompt "Link to products in the catalog." When I clicked "Link," the EAN code of my offer would become someone else's. However, my product was labeled with my own EAN code. So I was confused. If the EAN code of my offer changed, what should I do with the previous inventory?
